docs/csharp/whats-new/snippets/whats-new-csharp9/WeatherObservation.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NewCsharp9
+{
+    // <DeclareWeatherObservation>
+    public struct WeatherObservation
+    {
+        public DateTime RecordedAt { get; init; }
+        public decimal TemperatureInCelsius { get; init; }
+        public decimal PressureInMillibars { get; init; }
+
+        public override string ToString() =>
+            $"At {RecordedAt.ToShortTimeString()} on {RecordedAt.ToShortDateString()}: Temp = {TemperatureInCelsius}, with {PressureInMillibars} pressure";
```

Comment:
```suggestion
        public override string ToString() =>
            $"At {RecordedAt:h:mm tt} on {RecordedAt:M/d/yyyy}: " +
            $"Temp = {TemperatureInCelsius}, with {PressureInMillibars} pressure";
```

Concat string on new line for readability... trying to avoid horizontal scrolling, also used string interpolation format specifiers for date formatting to shorten it.